docs/doc/10-deploy/02-deploying-with-cloud-object-storage.md

```diff
@@ -0,0 +1,134 @@
+
+- Create a bucket or container named *databend*.
+- Get the endpoint URL for connecting to the bucket or container you created.
+- Get the Access Key ID and Secret Access Key for your account.
+
+:::tip
+
+**Need Help?** For information about how to manage buckets and Access Keys for your cloud object storage, refer to the user manual from the solution provider. Here are some useful links you may need:
+- Amazon S3
+  - https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html
+  - https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html
+
+- Tencent COS
+  - https://cloud.tencent.com/document/product/436/13309
+  - https://cloud.tencent.com/document/product/436/68282
+
+- Alibaba OSS
+  - https://www.alibabacloud.com/help/zh/object-storage-service/latest/create-buckets-2
+  - https://help.aliyun.com/document_detail/53045.htm
+
+- Wasabi
+  - https://wasabi.com/wp-content/themes/wasabi/docs/Getting_Started/index.html#t=topics%2FGS-Buckets.htm%23TOC_Creating_a_Bucketbc-1&rhtocid=_5_0
+  - https://wasabi.com/wp-content/themes/wasabi/docs/Getting_Started/index.html#t=topics%2FAssigning_an_Access_Key.htm
+
+- QingCloud QingStore
+  - https://docsv3.qingcloud.com/storage/object-storage/manual/console/bucket_manage/basic_opt/
+  - https://docs.qingcloud.com/product/api/common/overview.html
+
+- Azure Blob Storage
+  - https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal#create-a-container
+  - https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal#view-account-access-keys
+
+:::
+
+### Downloading Databend
+a. Create a folder named *databend* in the directory */usr/local*.
+
+b. Download and extract the latest Databend package for your platform from https://github.com/datafuselabs/databend/releases.
+
+c. Move the extracted folders *bin* and *etc* to the folder */usr/local/databend*.
+
+### Deploying a Meta Node
+a. Open the file *databend-meta-node.toml* in the folder */usr/local/databend/etc*, and replace *0.0.0.0* with *127.0.0.1* within the whole file.
+
+b. Open a terminal window and navigate to the folder */usr/local/databend/bin*.
+
+c. Run the following command to start the Meta node:
+
+```curl
+./databend-meta -c ../etc/databend-meta.toml > meta.log 2>&1 &
+```
+
+d. Run the following command to check if the Meta node was started successfully:
+
+```curl
+curl -I  http://127.0.0.1:28101/v1/health
+```
+
+### Deploying a Query Node
+a. Open the file *databend-query-node.toml* in the folder */usr/local/databend/etc*, and replace *0.0.0.0* with *127.0.0.1* within the whole file.
+
+b. In the file *databend-query-node.toml*, set the parameter type in [storage] block to s3 first. Then comment out the [storage.fs] block,  uncomment the [storage.s3] block and set your values in it.
+
+```toml
+[storage]
```

Comment:
We can add a tab config for each object storage to tell the user how to config the `endpoint_url`:
|Amazon S3|Tencent COS|Alibaba OSS|Wasabi|QingCloud QingStore|Azure Blob Storage

Like:
![image](https://user-images.githubusercontent.com/172204/176348477-a41ee341-e4cc-409f-8edd-71e60f18d6a2.png)